docsite/docs/api_pages/object-detection.md

```diff
@@ -20,24 +20,24 @@ For more details or to demo it, visit
 
 1. **Open Terminal or Command Prompt:** Open your terminal or command prompt application. Create a brand new React Native Project. In this case we will be using **'rnmpdemo'** as on React Native MediaPipe Demo for example.
 
-```jsx
+```
 npx react-native init rnmpdemo
 ```
 
 2. **Navigate to Your Project Directory:** Navigate to your React Native project directory on your environment.
 
 3. **Install React Native MediaPipe:** Run the following command to install React Native MediaPipe and its dependencies:
 
-```jsx
-yarn add react-native-mediapipe react-native-vision-camera react-native-worlets-core
+```
+yarn add react-native-mediapipe react-native-vision-camera react-native-worklets-core
 ```
 
-    If correctly you should see a **react-native-mediapipe** drowpdown inside **node_modules** 
+    If correctly you should see a **react-native-mediapipe** dropdown inside **node_modules** 
 
 
 4. **Configuring Babel:** Navigate to the 'babel.config.js' file and add:
 
-```jsx
+``` jsx
 module.exports = {
   presets: ['module:@react-native/babel-preset'],
   plugins: [['react-native-worklets-core/plugin']],
```

Comment:
Git doesn't let me add a comment in the right spot - this is for the next section, re configuring Gradle - that's only for Android and hence should go into the Android section.